server/server.ts

```diff
@@ -25,6 +107,23 @@ configureRoutes();
 app.listen(PORT, () => console.log('Example app listening at http://localhost:'+PORT))
 
 function configureRoutes() {
+  app.get('/login/facebook',
+    passport.authenticate('facebook')
+  );
+
+  app.get('/return',
+    passport.authenticate('facebook', { failureRedirect: '/' }),
+    function(req, res) {
+      res.redirect('/');
+  });
+
+  app.get('/api/me',
+    ensureLoggedIn,
+    function(req, res){
+      res.send({name: req.user.displayName});
```

Comment:
🛑  typing for user profile? currently displayName doesn't exist

http://www.passportjs.org/docs/profile/